src/crimson/osd/pg.cc

```diff
@@ -993,9 +993,9 @@ PG::with_locked_obc(Ref<MOSDOp> &m, const OpInfo &op_info,
     }
   case RWState::RWEXCL:
     if (oid.is_head()) {
-      return with_head_obc<RWState::RWWRITE>(oid, std::move(f));
+      return with_head_obc<RWState::RWEXCL>(oid, std::move(f));
```

Comment:
@xxhdx1985126 could you please create a PR including this change and the one of `excl_from_excl()` in another PR? so it can get merged sooner?